config/sync/core.entity_form_display.node.landing_page.default.yml

```diff
@@ -5,6 +5,7 @@ dependencies:
   config:
     - field.field.node.landing_page.body
     - field.field.node.landing_page.field_image
```

Comment:
Now that you have a hero image field, do you still need the other image field? Shouldn't this be removed?